test/integration/global-setup.js

```diff
@@ -0,0 +1,17 @@
+import { test as setup } from '@playwright/test'
+
+import Wiremock from '../testContainers/wiremock'
+import Localstack from '../testContainers/localstack'
+
+import config from '../../config/index'
+
+let localstack
+
+setup('Global setup', async () => {
+  setup.setTimeout(5 * 60 * 1000)
```

Comment:
need this here as it could take some time to download the images 
(5 mins is overkill, 2 mins should do. will do this as part of the next PR)

Reply:
Not sure about the Node binding, but usually Testcontainers SDKs provide a means to wait for the container to be ready, e.g. 

```
const container = await new GenericContainer("alpine")
  .withExposedPorts(6379)
  .withWaitStrategy(Wait.forListeningPorts())
  .start();
```

See https://node.testcontainers.org/features/wait-strategies/ for more info.

Reply:
looks like it would be useful to add a wait strategy, but we still need this so playwright doesn't time out. as playwright has a default timeout of 30 seconds for any one block